.vscode/extensions.json

```diff
@@ -25,7 +25,7 @@
 		// Description: An extension for setting-up multiple terminals at once, or just running some commands
 		// There are several Terminals defined in `.vscode/terminals.json` that can be used via this plugin.
 		"fabiospampinato.vscode-terminals",
-		// Name: Test Utils
+  // Name: Test Utils
 		// Description: Add, remove, and move .only in tests
 		"chrisbreiding.test-utils",
 		// Name: Toggle Quotes
```

Comment:
Bad merge conflict. Also, should be using spaces not tabs.

Reply:
Fixed